bindings/gumjs/gumquickprocess.c

```diff
@@ -182,6 +186,18 @@ gumjs_get_parent_module (GumQuickCore * core)
   return _gum_quick_core_load_module_data (core, "process");
 }
 
+GUMJS_DEFINE_GETTER (gumjs_process_get_main_module)
+{
+  GumQuickProcess * self;
+
+  self = gumjs_get_parent_module (core);
+
+  if (self->main_module == NULL)
+    self->main_module = gum_process_get_main_module ();
+
+  return _gum_quick_module_new (ctx, self->main_module, self->module);
```

Comment:
Perhaps we should cache the JS wrapper?

Reply:
done for quickjs, not super sure how to do it in v8, is there any example around?

Reply:
Yay! For V8, there's an example where we store it [here](https://github.com/frida/frida-gum/blob/ed4f17176f8671af93fbbb9afdbb584fb7825bf6/bindings/gumjs/gumv8interceptor.cpp#L394-L395), and then [use it](https://github.com/frida/frida-gum/blob/ed4f17176f8671af93fbbb9afdbb584fb7825bf6/bindings/gumjs/gumv8interceptor.cpp#L633-L634) at a later point.